third_party/move/move-model/bytecode/src/stackless_control_flow_graph.rs

```diff
@@ -297,9 +297,16 @@ impl std::fmt::Display for DotCFGEdge {
 }
 
 /// Generate the dot representation of the CFG (which can be rendered by the Dot program)
```

Comment:
nit: the stackless control flow graph has `backward` field in its representation, use that for consistency? 

nit: maybe we want a variant where `from_all_blocks` is false (for backward)?

Reply:
The argument is `func_target: &'env FunctionTarget<'env>` so we are actually generating the CFG and deciding its direction.